hw/top_earlgrey/formal/conn_csvs/ast_clkmgr.csv

```diff
@@ -0,0 +1,37 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Run these checks with:
+#  ./util/dvsim/dvsim.py hw/top_earlgrey/formal/chip_conn_cfg.hjson
+
+,NAME,SRC BLOCK,SRC SIGNAL,DEST BLOCK,DEST SIGNAL,,,,,,
+
+#################################
+# Signals from AST to Clkmgr
+#################################
+CONNECTION, AST_CLK_SYS_OUT,     u_ast, clk_src_sys_o,     top_earlgrey.u_clkmgr_aon,     clk_main_i
+CONNECTION, AST_CLK_AON_OUT,     u_ast, clk_src_aon_o,     top_earlgrey.u_clkmgr_aon,     clk_aon_i
+CONNECTION, AST_CLK_USB_OUT,     u_ast, clk_src_usb_o,     top_earlgrey.u_clkmgr_aon,     clk_usb_i
+CONNECTION, AST_CLK_IO_OUT,      u_ast, clk_src_io_o,      top_earlgrey.u_clkmgr_aon,     clk_io_i
+
+CONNECTION, AST_ALL_BYP_ACK_OUT, u_ast, all_clk_byp_ack_o, top_earlgrey.u_clkmgr_aon, all_clk_byp_ack_i
+CONNECTION, AST_IO_BYP_ACK_OUT,  u_ast, io_clk_byp_ack_o,  top_earlgrey.u_clkmgr_aon, io_clk_byp_ack_i
+
+#################################
+# Signals from Clkmgr to AST
+#################################
+# TODO: why is this clk_aon_secure in the chip-level, when the module itself is connected to clk_aon_peri?
+CONNECTION, AST_CLK_ADC_IN,     top_earlgrey.u_clkmgr_aon, clocks_o.clk_aon_peri,         u_ast, clk_ast_adc_i
+CONNECTION, AST_CLK_ALERT_IN,   top_earlgrey.u_clkmgr_aon, clocks_o.clk_io_div4_secure,   u_ast, clk_ast_alert_i
+CONNECTION, AST_CLK_ES_IN,      top_earlgrey.u_clkmgr_aon, clocks_o.clk_main_secure,      u_ast, clk_ast_es_i
+CONNECTION, AST_CLK_RNG_IN,     top_earlgrey.u_clkmgr_aon, clocks_o.clk_main_secure,      u_ast, clk_ast_rng_i
+# TODO: why is this clk_io_div4_secure in the chip-level when the peripheral Xbar runs on clk_io_div4_infra?
```

Comment:
@tjaychen for these two TODO's: both tests as formulated here pass since the clock manager connects these to the same clock net internally. But for consistency, would it make sense to align the connections in `chip_earlgrey_asic.sv`?

Reply:
we should change this from `clk_io_div4_secure` -> `clk_io_div4_infra`

Reply:
this is done.